dev-packages/cli/src/download-plugins.ts

```diff
@@ -73,7 +73,7 @@ export default async function downloadPlugins(options: DownloadPluginsOptions =
     const {
         packed = false,
         ignoreErrors = false,
-        apiVersion = '1.50.0',
+        apiVersion = '1.53.2', // should align with the framework.
```

Comment:
Could you add a little more detail to this comment about which things need to agree for this specification and the framework to be aligned? And maybe add a comment at each place so that anyone who changes one location knows they need to change others.

Reply:
@colin-grant-work I updated the docs to give additional information, please let me know if there was anything else you had in mind.